### PR TITLE
check off some items from the roadmap

### PR DESCRIPTION
**Pull request by BenTheElder**
/cc @amwat

### CODE REVIEW

**Comment by BenTheElder** [comment]
actuallly .. this passes verify anyhow, and doesn't touch code so not much point in the e2es.

**Comment by BenTheElder** [comment]
might be worth setting some run_if_changed, even though our tests are faster than k/k, they still take an awfully long time imho if they flake :|

File: site/content/docs/contributing/1.0-roadmap.md [hunk 1]
```diff
@@ -33,22 +33,26 @@ To reach "beta" [grade][deprecation-policy] kind needs to at minimum:
   - [x] add Windows binaries to releases - [#155]
   - [x] improve instructions for KUBECONFIG in particular
 - [ ] Support usage as a properly versioned, supported, and documented library. This includes following semver without every release being a major / breaking change to the API (which must be extensible without breakage), ensuring the CLI only uses a suitable public library surface, documentation and examples for the library, versioned types for public APIs (E.G. config format), etc.
-- [ ] should be possible to troubleshoot kind without needing to modify kind or use external debugging tools
-  - [ ] consistent logging (what is logged, when should it be logged, what levels are used)
-  - [ ] errors have appropriate context
+  - TODO: what exactly do we want here? Should this really be beta blocking?
+- [x] should be possible to troubleshoot kind without needing to modify kind ~or use external debugging tools~ (this should be possible now, if not perfect!)
+  - x ] consistent logging (what is logged, when should it be logged, what levels are used) (this is consistent-ish now, if not perfect)
```
Comment (tao12345666333) [inline site/content/docs/contributing/1.0-roadmap.md]: there is a small formatting error at the beginning of this line.

**Pull request merged**